api/Notification.json

```
@@ -735,7 +735,7 @@
               "version_added": null
             },
             "firefox": {
-              "version_added": false
+              "version_added": true
```

I did a little digging and it appears that this shipped along with most of the rest of the API in Firefox 22: https://bugzilla.mozilla.org/show_bug.cgi?id=782211

```suggestion
              "version_added": "22"
```

---

Good find! Looks good!